build.gradle.kts

```diff
@@ -16,34 +15,47 @@ repositories {
     maven { url = uri("https://repo.minebench.de/") }
     maven { url = uri("https://repo.extendedclip.com/content/repositories/placeholderapi/") }
     maven("https://repo.papermc.io/repository/maven-public/")
+    maven { url = uri("https://jitpack.io") }
 }
 
 dependencies {
     compileOnly("io.papermc.paper:paper-api:1.20.2-R0.1-SNAPSHOT")
-    compileOnly("me.xemor:UserInterface:1.6.6-SNAPSHOT")
     compileOnly("com.github.MilkBowl:VaultAPI:1.7")
     compileOnly("org.jetbrains:annotations:23.0.0")
     compileOnly("me.clip:placeholderapi:2.11.2")
-    shadow("net.kyori:adventure-text-minimessage:4.11.0")
-    shadow("net.kyori:adventure-platform-bukkit:4.1.1")
-    shadow("mysql:mysql-connector-java:8.0.29")
-    shadow("com.zaxxer:HikariCP:5.0.1")
-    shadow("org.xerial:sqlite-jdbc:3.36.0.3")
-    shadow("me.xemor:configurationdata:2.0.0-SNAPSHOT")
+    implementation("net.kyori:adventure-text-minimessage:4.11.0")
+    implementation("net.kyori:adventure-platform-bukkit:4.1.1")
+    implementation("mysql:mysql-connector-java:8.0.29")
+    implementation("com.zaxxer:HikariCP:5.0.1")
+    implementation("org.xerial:sqlite-jdbc:3.36.0.3")
+    implementation("com.github.Xemorr:ConfigurationData:537df1d205")
+    implementation("me.xemor:UserInterface:2.0.1-SNAPSHOT")
 }
 
 java {
-    configurations.compile.get().dependencies.remove(dependencies.gradleApi())
```

Comment:
Why did you remove this line out of interest? It was originally added because the entire gradle api was being shaded into plugins?

Reply:
or have you double checked that yours doesn't do this

Reply:
I used to have it in all my plugins and then stumbled into some issues at one point, iirc kotlin-dsl was what was causing the gradle api to be shaded in

Reply:
Could you whip out WinRAR / 7zip and see if it's being compiled into this?

Reply:
Comparing PlayerShopOverhaul to ActivityRewarder, I presume all the additional packages are drivers and stuff for adventure? No sign of the gradle packages though
![image](https://github.com/Xemorr/PlayerShopOverhaul/assets/55620516/2d542b5c-1b2b-4055-9afd-67a2f5f7b741)

Reply:
Great to hear, you're probably right.